.eslintrc.json

```diff
@@ -7,6 +7,7 @@
     "mocha": true
   },
   "rules": {
+    "object-curly-newline": 0,
```

Comment:
I was getting a false-positive with this and destructuring. We need to go through all these rules at some point